### PR TITLE
PivotTable MergedItem correct saving

### PR DESCRIPTION
**Pull request by b0bi79**
ClosedXML does not correctly save the MergedItem property of the PivotTable.

### CODE REVIEW

**Comment by igitur** [comment]
Sorry, but for anything pivot table related, you'll have to add complete unit tests. The pivot tables are too sensitive to avoid this.

**Comment by b0bi79** [comment]
Added a test and more fully tested the saving of the PivotTable options.

**Comment by b0bi79** [comment]
All defaults fixed according http://www.datypic.com/sc/ooxml/e-ssml_pivotTableDefinition.html

File: ClosedXML_Tests/Excel/PivotTables/XLPivotTableTests.cs [hunk 1]
```diff
@@ -29,5 +29,120 @@ public void PivotTables()
...
+                        Assert.AreEqual(true, ptassert.ShowEmptyItemsOnRows, "ShowEmptyItemsOnRows save failure");
+                        Assert.AreEqual(true, ptassert.ShowEmptyItemsOnColumns, "ShowEmptyItemsOnColumns save failure");
+                        Assert.AreEqual(false, ptassert.DisplayItemLabels, "DisplayItemLabels save failure");
+                        Assert.AreEqual(true, ptassert.SortFieldsAtoZ, "SortFieldsAtoZ save failure");
+                        Assert.AreEqual(true, ptassert.PrintExpandCollapsedButtons, "PrintExpandCollapsedButtons save failure");
+                        Assert.AreEqual(true, ptassert.RepeatRowLabels, "RepeatRowLabels save failure");
+                        Assert.AreEqual(true, ptassert.PrintTitles, "PrintTitles save failure");
+                        Assert.AreEqual(false, ptassert.SaveSourceData, "SaveSourceData save failure");
+                        Assert.AreEqual(false, ptassert.EnableShowDetails, "EnableShowDetails save failure");
+                        // TODO Assert.AreEqual(false, ptassert.RefreshDataOnOpen, "RefreshDataOnOpen save failure");
```
Comment (igitur) [inline ClosedXML_Tests/Excel/PivotTables/XLPivotTableTests.cs]: What's the problem with this Assert?

Comment (b0bi79) [reply ClosedXML_Tests/Excel/PivotTables/XLPivotTableTests.cs]: You mean TODO? It is commented out for ptassert.RefreshDataOnOpen. 
RefreshDataOnOpen must be implemented.

**Pull request merged**